sync-wp/components/config/screen.py

```
@@ -0,0 +1,105 @@
+from PySide6.QtWidgets import (QWidget, QVBoxLayout,
+                              QFormLayout, QGroupBox, QLabel, 
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)